src/playlist.js

```diff
@@ -0,0 +1,66 @@
+var playItem = require('./playitem.js');
+var setupAutoadvance = require('./autoadvance.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var autoadvanceTimeout = null;
+
+  var playlist = function playlist(list) {
+    if (typeof list === 'number') {
+      return player.playlist.item(list);
```

Comment:
I have a feeling I know what your answer will be, but that's exactly how jQuery plugins work, though even more obviously with `jQuery.fn == jQuery.prototype`.

>  I'm more than happy with the .item() method call
playlist.item() or player.item()